Source/UZKFileInfo.m

```diff
@@ -105,4 +107,20 @@ - (NSDate *)readDate:(tm_unz)date
     return [calendar dateFromComponents:components];
 }
 
+- (BOOL)isDirectoryWith:(unz_file_info64 *)fileInfo {
+    
+    uLong type = fileInfo->external_fa >> 0x1D & 0x1F;
+    if (0 == (fileInfo->version >> 8)) {    //is DOS-archive
+        type = fileInfo->external_fa >> 4;
```

Comment:
- [ ] Is this really a type? I think it's the MS-DOS format's directory attribute byte, and should have a new variable definition with a name describing exactly what it is
- [ ] I'd like to have a test case that goes into this `if` block to test how it works for MS-DOS format archives
- [ ] Once that's all done, the comment won't be necessary, because the condition will be something like `os == ZipOSMSDOS`, which is self-documenting

Reply:
This is the judgment of the MS-DOS type.
我想重新修改这些粗糙的代码，并新的 2.0重新提交。 我想关闭这个 PR，并在完成之后重新提交。

Reply:
his is the judgment of the MS-DOS type.
I want to re-edit these rough code and re-submit the new 2.0. I want to close this PR and resubmit it when it's done.